### PR TITLE
[CR-1435] Demo page to show Checkout and SDK

### PR DESCRIPTION
**Pull request by Arlanthir**
Video: https://www.loom.com/share/e03a698a68f84ab99ec91c3436254dc1

For now it's 100% basic, I'm torn between leaving it as is to focus on the necessary integration or adding some fluff like page structure and the outside customer form.

Thoughts?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  "main": "app.js",
+  "scripts": {
+    "dev": "node app.js",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "easypay",
+  "license": "ISC",
+  "dependencies": {
+    "dotenv": "14.2.0",
+    "easypay-checkout": "file:../checkout-sdk",
```
Comment (Espina2) [inline package.json:14]: Didn't know that we can do this magic. :+1:

**Comment by Espina2** [comment]
I video would be appreciated to see the look and feel of the thing :)

File: package.json (hunk 1 above)
Comment (Arlanthir) [reply package.json:14]: It symlinks it in the node_modules, which can create problems of its own. We need to test it properly.

**Comment by Arlanthir** [comment]
Removed start_time after realizing it wasn't required.

**Review by Espina2 — APPROVED** [review]

**Pull request merged**